src/sys/socket/mod.rs

```diff
@@ -32,32 +32,28 @@ pub mod sockopt;
 
 pub use self::addr::{SockaddrLike, SockaddrStorage};
 
-#[cfg(not(any(target_os = "illumos", target_os = "solaris")))]
-#[allow(deprecated)]
-pub use self::addr::{AddressFamily, SockAddr, UnixAddr};
 #[cfg(any(target_os = "illumos", target_os = "solaris"))]
 #[allow(deprecated)]
```

Comment:
Do we still need the `#[allow(deprecated)]` attribute here and below?

Reply:
Removed. Not quite sure why they were suppressing deprecations anyway.

Reply:
Thanks.  I think they were there because `pub use` counts as "using" a deprecated item, even though it's from the same crate.